build.gradle

```diff
@@ -63,7 +65,17 @@ dependencies {
 test {
     scanForTestClasses false
     include "**/*Test.class"
-    exclude "**/*IntegrationTest*"
+    exclude "**/*IntegrationTest*", "**/*PypiScannerTest*"
+    testLogging {
+        exceptionFormat "full"
+        events "started", "passed", "skipped", "failed", "standardOut", "standardError"
+        minGranularity 0
+    }
+}
+
+tasks.register('pythonTests', Test) {
```

Comment:
Now that we have "tests", "integrationTests" and "pythonTests" I think it is worth mentioning them in the README or in a new CONTRIBUTION.md file